optimum/habana/transformers/trainer.py

```diff
@@ -53,6 +54,7 @@
     nested_numpify,
     nested_truncate,
     reissue_pt_warnings,
+    is_sagemaker_mp_enabled,
```

Comment:
You can remove this, we don't need it in Optimum Habana

Reply:
removed